src/spawn.cpp

```diff
@@ -336,6 +333,12 @@ bool Spawn::spawnMonster(uint32_t spawnId, MonsterType* mType, const Position& p
 	monster->setSpawn(this);
 	monster->setMasterPos(pos);
 	monster->incrementReferenceCounter();
+	if (!g_events->eventMonsterOnSpawn(monster, pos, startup, false)) {
```

Comment:
not a fan of this, placing a monster to delete it, sounds like a waste

Reply:
Then return false won't be fixed. 🤷‍♂️